Sources/Extensions/Foundation/UserDefaultsExtensions.swift

```diff
@@ -37,5 +37,33 @@ public extension UserDefaults {
 	public func date(forKey key: String) -> Date? {
 		return object(forKey: key) as? Date
 	}
+    
+    /// SwifterSwift: Retrieves a Codable object from UserDefaults.
+    ///
+    /// - Parameters:
+    ///   - type: Class that conforms to the Codable protocol.
+    ///   - key: key to find the Codable object for.
+    /// - Returns: Codable object for key (if exists).
+    public func object<T: Codable>(_ type: T.Type, with key: String) -> Codable? {
+        if let data = UserDefaults.standard.value(forKey: key) as? Data {
```

Comment:
Hey @jason-ingenuity, since you are defining this as a UserDefaults exetension, you don't need to call the shared instance like `UserDefaults.standard.value(forKey: key)`, you can just call the instance method directly `self.value(forKey: key)` :)

Reply:
Yes, I overlooked it. Thanks.